src/modm/driver/radio/sx128x_impl.hpp

```diff
@@ -0,0 +1,412 @@
+/*
+ * Copyright (c) 2023, Lasse Alexander Jensen
+ * Copyright (c) 2023, Rasmus Kleist Hørlyck Sørensen
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef MODM_SX128X_HPP
+#   error "Don't include this file directly, use 'sx128x.hpp' instead!"
+#endif
+
+namespace modm
+{
+
+// ----------------------------------------------------------------------------
+
+template < class Hal >
+Sx128x< Hal >::Sx128x()
+{
+}
```

Comment:
Don't think this is necessary.

Reply:
I have changed it to use the default constructor for now